apps/screener-proxy/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "screener-proxy",
+  "version": "1.0.0",
+  "description": "",
+  "private": true,
+  "scripts": {
+    "build": "tsc",
+    "start": "node dist/index",
+    "dev": "nodemon index.ts"
+  },
+  "dependencies": {
+    "@octokit/app": "10.1.0",
+    "body-parser": "^1.19.0",
+    "express": "4.17.1"
```

Comment:
Please use the same version of `express`, `@types/express`, and `@types/node` as `scripts is using. (or in the case of express you can update the versions everywhere if you want)